build.maven/ivy.project/pom.xml

```diff
@@ -1,13 +1,11 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
 	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
 	<modelVersion>4.0.0</modelVersion>
-	<groupId>com.axonivy.ivy.supplements</groupId>
 	<artifactId>ivy-project</artifactId>
-	<version>7.4.0-SNAPSHOT</version>
```

Comment:
Also raise build-plugin version (project-build-plugin.version) and minimum Engine version (ivyVersion) further down.